test/zdtm/static/shmemfd.desc

```diff
@@ -0,0 +1 @@
+{'flavor': 'h', 'flags': 'suid'}
```

Comment:
Can we also add ns flavour?

Reply:
We can't do this, because the test opens /proc/self/map_files.

Reply:
You can't open map_files in uns, but in ns flavour test passed for me.

Reply:
yes, you are right. Fixed.